website/community/community.md

```diff
@@ -0,0 +1,19 @@
+# Community
+
+We have setup the following regular syncs for community users and developers to meet, interact and exchange ideas. Meetings will be recorded and made available after every sync.
+We have a slack channel [link] you can join for asking any clarification questions.
+
+## Bi-weekly Community Call
```

Comment:
@dipankarmazumdar Need your help in improving the content for this page, you can checkout the branch and add links for slack etc.